generators/Generators.csproj

```diff
@@ -15,4 +15,9 @@
   <ItemGroup>
     <EmbeddedResource Include="Output\Templates\*.liquid" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@ErikSchierboom yeah I'm not happy with this. 

I just came back to this and it looks like my red x's are back, even with this change. I'll need to dig into the root cause some more to find out why these are popping up.